src/routes/exercises/components/LockingLane.tsx

```diff
@@ -14,7 +14,7 @@ const Container = styled.View`
 
 const Line = styled.View<{ color: string }>`
   width: 2px;
-  height: ${hp('5%')}px;
+  height: ${hp('6%')}px;
   background-color: ${props => props.color};
   margin: ${props => props.theme.spacings.xs} 10px;
```

Comment:
(optional) i just saw that there is a random margin(left/right) which should not be set instead use `align-self: center` for horizontal placement of the Line.
This is optional since it was not your change ;)

Reply:
Oh right, I remember there is an open issue for this. Should I fix it here or leave all of these kind of changes to that issue?

Reply:
Link in a comment or link in this conversation?

Reply:
link in this conversation :)

Reply:
My bad, ther is no LUN issue yet, but I assume the same reasoning as for IGAPP-1038 applies?

Reply:
Not really I think since Lunes is only LTR and does not have any RTL languages. Also, this is not really direction dependent since the same margin is used left and right.

So maybe you could fix it here after all (optional) or just keep it the way it is. We probably don't have much other occurrences.